site/content/en/docs/installation/_index.md

```diff
@@ -259,7 +259,7 @@ The currently supported features are:
 | `ConfigurableResourceTransformations` | `false` | Alpha      | 0.9   | 0.9   |
 | `ConfigurableResourceTransformations` | `true`  | Beta       | 0.10  |       |
 | `WorkloadResourceRequestsSummary`     | `false` | Alpha      | 0.9   | 0.9   |
-| `WorkloadResourceRequestsSummary`     | `true`  | Beta       | 0.10  |       |
+| `WorkloadResourceRequestsSummary`     | `true`  | Beta       | 0.10  | 0.11  |
```

Comment:
```suggestion
| `WorkloadResourceRequestsSummary`     | `true`  | GA    | 0.10  | 0.11  |
```

Reply:
I changed to conform to how `MultiplePreemptions` is listed for a similar lifecycle..